src/styles/style.scss

```diff
@@ -0,0 +1,9 @@
+// Base
+@import "general";
+
+// Compositions
+@import "blocks/table";
+@import "blocks/tabs";
```

Comment:
We briefly talked about this in our quick call today - I think it would be nicer to name these `components` instead of `blocks` since we are using components everywhere else.